libevmasm/Assembly.cpp

```diff
@@ -117,7 +117,7 @@ string locationFromSources(StringMap const& _sourceCodes, SourceLocation const&
 	if (_location.isEmpty() || _sourceCodes.empty() || _location.start >= _location.end || _location.start < 0)
 		return "";
 
-	auto it = _sourceCodes.find(*_location.sourceName);
+	auto it = _sourceCodes.find(_location.source->name());
```

Comment:
Please protect against `_location.source` being null - I know it was not there before, but don't let bad legacy code stand in our way :)

Reply:
done 2 lines above (now). thx.